roles/openshift_hosted/README.md

```diff
@@ -33,13 +33,20 @@ variables also control configuration behavior:
 
 | Name                                         | Default value | Description                                                                  |
 |----------------------------------------------|---------------|------------------------------------------------------------------------------|
-| openshift_hosted_registry_glusterfs_swap     | False         | Whether to swap an existing registry's storage volume for a GlusterFS volume |
-| openshift_hosted_registry_glusterfs_swapcopy | True          | If swapping, also copy the current contents of the registry volume           |
+| openshift_hosted_registry_storage_glusterfs_swap     | False         | Whether to swap an existing registry's storage volume for a GlusterFS volume |
+| openshift_hosted_registry_storage_glusterfs_swapcopy | True          | If swapping, also copy the current contents of the registry volume           |
+| openshift_hosted_registry_storage_glusterfs_ips      | []            | If we use external gluster for Registry backend we need to provide list of IP's for EP creation|
+| openshift_hosted_registry_storage_glusterfs_path     | []            | If we use external gluster we need to provide volume name from gluster cluster for mounting|
+
+**Note:** if you have host group `glusterfs_registry` set in your invetory, variable `openshift_hosted_registry_storage_glusterfs_ips`, CAN'T be used because this is 2 different use-cases - CNS/CRS vs External Native Gluster. 
+_
 
 Dependencies
 ------------
 
 * openshift_hosted_facts
+* openshift_storage_glusterfs
```

Comment:
This isn't true. It's not a dependency if you're not using GlusterFS storage. Remove it.